src/api/integrations/companies-house/index.ts

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios';
+import dotenv from 'dotenv';
+import { CompanyHouseAPIResponse } from '../../types';
+
+dotenv.config();
+
+const username: any = process.env.COMPANIES_HOUSE_API_KEY;
+const companiesHouseURL: any = process.env.COMPANIES_HOUSE_API_URL;
+
+/**
+ * getCompanyHouseDetails
+ * makes companies house API call and returns company
+ * @param {String} companyNumber
+ * @returns {CompanyHouseAPIResponse} CompanyHouseResponse object
+ */
+const companiesHouse = {
+  getCompanyHouseDetails: async (companyNumber: string): Promise<CompanyHouseAPIResponse> => {
+    try {
+      const response = await axios({
+        method: 'get',
+        url: `${companiesHouseURL}/company/${companyNumber}`,
+        auth: { username, password: '' },
```

Comment:
is empty password required?

Reply:
yes it is required as part of the auth types